### PR TITLE
Added support for passing kernel options to preseed via templating

### PR DESCRIPTION
**Pull request by BaptisteGerondeau**
This fixes issue #62

### CODE REVIEW

**Comment by gemagomez** [comment]
Please, fix lint issue and add a comment to CHANGELOG.md about this change.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Bug fixes:
 
  - ui: set `<title>` for console page based on current machine name
 
+Features:
+
+ - preseed: added support for kernel options command line to be passed to the preseed, via kernel_optionsjinja2 variable, implementing Issue #62's request.
```
Comment (niedbalski) [inline CHANGELOG.md]: missing space

**Comment by niedbalski** [comment]
LGTM +1

**Comment by gemagomez** [comment]
LGTM - thanks!

**Pull request merged**